this & object prototypes/ch2.md

```diff
@@ -380,36 +380,36 @@ Internally, these various functions almost certainly use *explicit binding* via
 
-First, let's re-define what a "constructor" in JavaScript is. In JS, constructors are **just functions** that happen to be called with the `new` operator in front of them. They are not attached to classes, nor are they instantiating a class. They are not even special types of functions. They're just regular functions that are, in essence, hijacked by the use of `new` in their invocation.
+Primeiro, vamos redefinir o que é um "construtor" em JavaScript. Em JS, construtores são **apenas funções** que são chamadas quando o operador `new` está na frente delas. Elas não são vinculadas à classes, nem estão instanciando-as. Elas não são nem tipos especiais de funções. Elas são apenas funções normais que são, em essência, sequestradas pelo uso de `new` quando invocadas.
 
-For example, the `Number(..)` function acting as a constructor, quoting from the ES5.1 spec:
+Por exemplo, a função `Number(..)` atua como um construtor, citação da especificação ES5.1:
 
-> 15.7.2 The Number Constructor
+> 15.7.2 O Construtor Number
 >
-> When Number is called as part of a new expression it is a constructor: it initialises the newly created object.
+> Quando Number é chamado como parte de uma expressão ele é um construtor: ele inicializa o novos objetos criados.
 
-So, pretty much any ol' function, including the built-in object functions like `Number(..)` (see Chapter 3) can be called with `new` in front of it, and that makes that function call a *constructor call*. This is an important but subtle distinction: there's really no such thing as "constructor functions", but rather construction calls *of* functions.
+Então, praticamente qualquer função antiga, incluindo as funções internas do objeto, como `Number(..)` (Veja o Capítulo 3) podem ser chamadas com `new` à frente, e isso faz dessa chamada uma *chamada de construtor*. Essa é uma importante mas sútil diferença: não há realmente nenhuma coisa como "funções construtoras", mas sim chamadas de *de funções* construtoras.
 
-When a function is invoked with `new` in front of it, otherwise known as a constructor call, the following things are done automatically:
+Quando uma função é invocada com `new` à sua frente, também conhecida como chamada de construtor, as seguintes coisas são feitas automaticamente:
 
-1. a brand new object is created (aka, constructed) out of thin air
-2. *the newly constructed object is `[[Prototype]]`-linked*
-3. the newly constructed object is set as the `this` binding for that function call
-4. unless the function returns its own alternate **object**, the `new`-invoked function call will *automatically* return the newly constructed object.
+1. um objeto novo em folha é criado (construído) fora do ar
```

Comment:
Acho que "novo em folha" ficou ótimo como tradução para "out of thin air", mas eu removeria o `fora do ar` :)